src/winetricks

```diff
@@ -12413,6 +12413,84 @@ load_pngfilt()
 
 #----------------------------------------------------------------
 
+w_metadata powershell_core apps \
```

Comment:
Sorry I missed this before, but it's probably better to use 'dlls' here.

Reply:
fixed with https://github.com/Winetricks/winetricks/pull/2211/commits/768e913101d419be762a93c142e2814808179d8d